### PR TITLE
Develop to Main Merge - Release

### PR DESCRIPTION
**Pull request by elima-sugunan**
# Problem:

# Solution:

# Testing

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,14 @@
+#
+# Copyright (c) 2021, 2022 by Delphix. All rights reserved.
+#
+
```
Comment (nick-mathison) [inline CONTRIBUTING.md:4]: We decided that we would not be accepting contributions.

Comment (nick-mathison) [reply CONTRIBUTING.md:4]: From my perspective, I don't mind having this section. But I recognize that there is a concern of supporting/managing it from development.

Comment (nick-mathison) [reply CONTRIBUTING.md:4]: Here is an example of Terraform which denies contributions: https://github.com/delphix-integrations/terraform-provider-delphix

File: README.md [hunk 2]
```diff
@@ -123,9 +112,9 @@ Please read [CONTRIBUTING.md](https://github.com/delphix/.github/blob/master/CON
 We use [SemVer](http://semver.org/) for versioning. For the versions available, see the [tags on this repository](https://github.com/your/project/tags).
```
Comment (nick-mathison) [inline README.md:112]: The `Contributing` section of the readme also needs to be updated.

File: CONTRIBUTING.md (hunk 1 above)
Comment (rohitdelphix) [reply CONTRIBUTING.md:4]: Added the same in "CONTRIBUTING.md"

Comment (elima-sugunan) [reply CONTRIBUTING.md:4]: Added the statement "This project is currently not accepting external contributions." in Contributing.md.

File: README.md (hunk 2 above)
Comment (elima-sugunan) [reply README.md:112]: Mentioned "This project is currently not accepting external contributions."

**Review by rohitdelphix — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -146,4 +135,5 @@ License
  See the License for the specific language governing permissions and
  limitations under the License.
  ```
-Copyright (c) 2014, 2016 by Delphix. All rights reserved.
+
+Copyright (c) 2014, 2022 by Delphix. All rights reserved.
```
Comment (gmanhas23) [inline README.md]: I think this could go on top same as in other files

**Review by gmanhas23 — APPROVED** [review]
LOoks good to me. Just one change in moving Copyright line to the top of the file in README.md

File: README.md (hunk 3 above)
Comment (rohitdelphix) [reply README.md]: It comes as a very large heading in README. Do we follow the same for all other README's too ?

Comment (rohitdelphix) [reply README.md]: DB2 adds it but as a comment like below

``

Comment (rohitdelphix) [reply README.md]: But then it is not visible in the preview

Comment (rohitdelphix) [reply README.md]: Followed terraform and mySQL and removed the copyright.

**Review by rohitdelphix — APPROVED** [review]

**Pull request merged**